styles/pypi/catppuccin.user.css

```diff
@@ -281,9 +288,13 @@
       text-decoration-color: @accent-color !important;
     }
 
+    .sponsors {
+      display: none;
+    }
+
     .package-snippet {
       background-color: @crust !important;
-      border: none !important;
+      border-color: invisible !important;
```

Comment:
`border-color: invisible` is not a valid value.

Reply:
fixed